zng/docs/spec.md

```diff
@@ -537,19 +568,19 @@ grammar describing the textual type encodings is:
 
 <alias-name> := <id>
 
-<id> := <id_start> <id_continue>*
+<id> := <id-start> <id-continue>*
 
-<id_start> := [A-Za-z_$]
+<id-start> := [A-Za-z_$]
 
-<id_continue> := <id_start> | [0-9]
+<id-continue> := <id-start> | [0-9]
```

Comment:
I noticed that the majority of our other `<things-in-angle-brackets>` use hyphens instead of underscores, so I've made that consistent here.